music_assistant/providers/filesystem_local/__init__.py

```diff
@@ -1028,7 +1028,7 @@ async def _get_local_images(self, folder: str) -> UniqueList[MediaItemImage]:
                 if item.ext != ext:
                     continue
                 # try match on filename = one of our imagetypes
-                if item.name in ImageType:
+                if item.name in ImageType.__members__:
```

Comment:
```suggestion
                if item.name in ImageType:
```
any reason for doing this as this should not be needed. ImageType is a StrEnum

Reply:
![Scherm­afbeelding 2024-11-20 om 18 47 28](https://github.com/user-attachments/assets/b5cafbc8-7ef6-4cd2-a037-76ccab33fc6a)

Reply:
The pyproject.toml suggests that Python 3.11 is still supported, but that does not work in 3.11:


```python
❯ python
Python 3.11.9 (main, Apr  2 2024, 08:25:04) [GCC 13.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from music_assistant_models import enums
>>> enums.ImageType
<enum 'ImageType'>
>>> "abc" in enums.ImageType
<stdin>:1: DeprecationWarning: in 3.12 __contains__ will no longer raise TypeError, but will return True or
False depending on whether the value is a member or the value of a member
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/run/current-system/sw/lib/python3.11/enum.py", line 742, in __contains__
    raise TypeError(
TypeError: unsupported operand type(s) for 'in': 'str' and 'EnumType'
>>>
```

Reply:
ah.. we're using 3.12 in our base images so lets bump that